hazelcast/include/hazelcast/client/protocol/ClientMessage.h

```diff
@@ -1207,6 +1211,10 @@ class HAZELCAST_API ClientMessage
         header->flags = is_final ? IS_FINAL_FLAG : DEFAULT_FLAGS;
         std::memcpy(
           fp + SIZE_OF_FRAME_LENGTH_AND_FLAGS, &bytes[0], bytes.size());
+
+        copy(begin(value.schemas_will_be_replicated()),
```

Comment:
Would it be better to get the value.schemas_will_be_replicated() as auto and put it to begin and end, instead of calling the function twice?

Reply:
Fixed.